### PR TITLE
Github-Action-Code-Quality

### PR DESCRIPTION
**Pull request by georgioupanayiotis**

### CODE REVIEW

File: .github/workflows/code-scan.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: Code Quality and Security Scanning
```
Comment (thepurpleowl) [inline .github/workflows/code-scan.yml]: @georgioupanayiotis Please change the action description to only CodeQL analysis

File: .github/workflows/code-scan.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        run: |
+          mkdir build
+          cd build
+          cmake ..
+          make
+
+      - name: Perform CodeQL analysis
+        uses: github/codeql-action/analyze@v2
+
+  lint_java:
```
Comment (thepurpleowl) [inline .github/workflows/code-scan.yml]: Let's remove linting jobs, as linting alerts are already part of project [codacy reports](https://app.codacy.com/gh/thepurpleowl/House-of-Codes/issues/current)

Comment (georgioupanayiotis) [reply .github/workflows/code-scan.yml]: removed

**Comment by georgioupanayiotis** [comment]
@thepurpleowl changes added can you please merge the PR and add label `hacktoberfest-accepted`

**Pull request merged**